Source/Scene/Cesium3DTileFeature.js

```diff
@@ -41,12 +45,21 @@ define([
      *     }
      * }, Cesium.ScreenSpaceEventType.MOUSE_MOVE);
      */
-    function Cesium3DTileFeature(tileset, content, batchId) {
+    function Cesium3DTileFeature(tileset, content, batchId, billboardCollection, labelCollection) {
```

Comment:
These changes make `Cesium3DTileFeature` really heavy especially for all the non-vector features that do not need most of these properties.  Is it possible to cleanly add a `Cesium3DTileVectorFeature` so we only pay this cost when needed?  Is it also possible to amortize some properties over multiple features, e.g., do all features with the same batch table also have the same billboard and label collection?